helm/aphw-ddi-events/values.yaml

```diff
@@ -37,6 +37,9 @@ container:
   azureStorageConnectionString: dummy
   azureStorageUseConnectionString: false
   azureStorageAccountName: dangerousdogindex
+  publicKeyPortal: dummy
+  publicKeyEnforcement: dummy
+  permittedDomains: "[]"
```

Comment:
Not sure if the pipeline likes double-quotes in the yaml. Don;t think I've used them before so might just need to see if ti gets through